lib/rectangle.js

```diff
@@ -0,0 +1,26 @@
+function area(rectangle) {
+  return rectangle.width * rectangle.height
+}
+
+function perimiter(rectangle) {
+  return (rectangle.width * 2) + (rectangle.height * 2)
+}
+
+function diagonal(rectangle) {
+  return Math.sqrt(Math.pow(rectangle.width, 2) + Math.pow(rectangle.height, 2))
+}
+
+function isSquare(rectangle) {
+  if (rectangle.width === rectangle.height) {
```

Comment:
Since you are returning a boolean either way, you can just return the expression in your `if` line.  The way you have it now, you're basically saying "If true, return true. If false return false."